docs/quick-start/sysctl-daemonset.yaml

```diff
@@ -0,0 +1,39 @@
+# Apply sysctl configuration required by Elasticsearch
+#
+# This DaemonSet will re-run sysctl every 60s on all nodes.
+#
+# XXX See CronJob daemonset which will allow scheduling one-shot or repeated
+# jobs across nodes:
+# https://github.com/kubernetes/kubernetes/issues/36601
+
+apiVersion: "extensions/v1beta1"
+kind: "DaemonSet"
+metadata:
+  name: "navigator-elasticsearch-sysctl"
+  namespace: "kube-system"
+spec:
+  template:
+    metadata:
+      labels:
+        app: "navigator-elasticsearch-sysctl"
+    spec:
+      containers:
+      - name: "apply-sysctl"
+        image: "busybox:latest"
+        resources:
+          limits:
+            cpu: "10m"
+            memory: "8Mi"
+          requests:
+            cpu: "10m"
+            memory: "8Mi"
+        securityContext:
+          privileged: true
+        command:
+        - "/bin/sh"
+        - "-c"
+        - |
+          set -o errexit
+          set -o xtrace
+          sysctl -w vm.max_map_count=262144
+          sleep 60s
```

Comment:
We need to actually loop in here when applying the value, else Kubernetes will eventually put the pod into a CrashLoopBackoff state

Reply:
Yeah, I thought about that, but wasn't sure if there'd be CrashLoopBackoff if the process exits with zero. I'll add the loop.

See:
 * https://github.com/kubernetes/kubernetes/issues/58619
 * https://github.com/kubernetes/kubernetes/issues/50962
 * https://github.com/kubernetes/kubernetes/issues/53125